modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/exception/FormFieldLimitException.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.exception;
+
+import com.liferay.portal.kernel.exception.FieldLimitException;
+
+/**
+ * @author Lázaro Costa
+ */
+public class FormFieldLimitException extends FieldLimitException {
```

Comment:
@lazhcsilva instead of creating this class, you must change the [service.xml](https://github.com/carolmariaabb/liferay-portal/blob/859bd92642d77fd39ebfb3fb8815b862ee9d8145/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/service.xml#L804-L812). This class will be generated after running `gw buildService` inside the `dynamic-data-mapping-service` folder.